src/pages/configeditor/migrating_data.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Configuration Editor Migration"
+excerpt: "Configuration Editor Migration."
+categories: configeditor
+slug: migration
+toc: true
+---
+
+## Migrating Early Access Program Data
+
+Customers that were part of the Early Access Program (EAP) can follow these steps to migrate existing artifacts to the newly installed Merative ACD Configuration Editor instace.
+
+1.  Stop the Merative Configuration Editor from the `acd-ce` diretory.
```

Comment:
typo:  should be "directory"
Suggestion re-phrase the sentence:  From the `acd-ce` directory, stop the Merative Configuration Editor.